packages/admin-ui/src/pages/wifi/components/localNetwork/LocalNetworkStatus.tsx

```diff
@@ -0,0 +1,111 @@
+// React
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+// API
+import { api, useApi } from "api";
+// Own components
+import { confirm } from "components/ConfirmDialog";
+import { withToast } from "components/toast/Toast";
+import ErrorView from "components/ErrorView";
+import Loading from "components/Loading";
+import Card from "components/Card";
+import Switch from "components/Switch";
+import { getDappnodeIdentityClean } from "services/dappnodeStatus/selectors";
+import { avahiFriendlyName, avahiLocalDomain } from "params";
+import { StateBadgeAvahiDaemon } from "pages/packages/components/StateBadge/StateBadgeAvahiDaemon";
+import { MdWifi } from "react-icons/md";
+
+function AvahiInfo() {
+  const dappnodeIdentity = useSelector(getDappnodeIdentityClean);
+  return (
+    <p>
+      Use the Local Network Discovery service and connect to your DAppNode by
+      connecting your device to the same router as your DAppNode. Go to{" "}
+      <a href={avahiLocalDomain}>http://my.dappnode.local</a>
+      {dappnodeIdentity.internalIp === dappnodeIdentity.ip &&
+        "Local and public IPs are equal. This may be due to dappnode is running on a remote machine and does not require Local Network Discovery."}
+    </p>
```

Comment:
Try to make this text consistent with the WIFI description. I would structure it this way (conceptually)

```
brief sentence describing        link to docs

link to UI for usage
```